.github/workflows/benches.yml

```diff
@@ -23,26 +24,31 @@ jobs:
           access_token: ${{ secrets.GITHUB_TOKEN }}
 
   bench:
-    name: All - Linux
-    # Full CI suites for this platform were only recently introduced.
-    # Some failures are permitted until we can properly correct them.
-    continue-on-error: true
+    name: Bench - Linux
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
       - run: make ci-sweep
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      - name: Download baseline benchmarks
+        uses: dawidd6/action-download-artifact@v2
```

Comment:
In https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions Github suggests

> Pin actions to a full length commit SHA
>
> Pinning an action to a full length commit SHA is currently the only way to use an action as an immutable release. Pinning to a particular SHA helps mitigate the risk of a bad actor adding a backdoor to the action's repository, as they would need to generate a SHA-1 collision for a valid Git object payload.

Could we also do this?

Reply:
Good idea :+1: